decidim-assemblies/spec/system/filter_assemblies_spec.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Filter Assemblies", type: :system do
```

Comment:
Nice one moving these tests to another file! 👏 👏 👏 👏

Reply:
Thanks